VMEncryption/EV2/ExtensionInfo.Public.xml

```diff
@@ -0,0 +1,31 @@
+<ExtensionInfo>
+    <Namespace>Microsoft.Azure.Security</Namespace>
+    <Type>AzureDiskEncryptionForLinux</Type>
+    <ExtensionShortName>AdeLinux</ExtensionShortName>
+    <Version>TO BE SET AT BUILDTIME</Version>
+    <SupportedOS>Linux</SupportedOS>
+    <ExtensionLabel>Windows Azure VMEncryption Extension for Linux IaaS</ExtensionLabel>
```

Comment:
Suggest changing "Windows Azure VMEncryption Extension for Linux IaaS" to "Azure Disk Encryption For Linux"